ruby/guessing-game/guessing_game.rb

```diff
@@ -0,0 +1,47 @@
+require 'io/console'
+
+class Game
+  attr_reader :guesses, :word, :line, :guessed_arr, :line_arr
+  def initialize(word)
+    @word = word.split("")
+    @guesses = word.length * 2
+    @line_arr = @word.map {|x| x = "_"}
+    @line = @line_arr.join(" ")
+    @guessed_arr = []
+    @replace_arr = []
+  end
+
+  def make_a_guess(letter)
+    if @guessed_arr.include?(letter)
+      p "You've already guessed that."
+    else
+      if @word.include?(letter)
+        @replace_arr = @word.each_index.select{|i| @word[i] == letter}
```

Comment:
If you follow my suggestion above we wont be reassigning the replace array every time you run this guess method.

Instead we could just move on to evaluation and completion.  The game could end if `@word == @replace_arr.join('')`